lib/util/git/index.ts

```diff
@@ -475,6 +474,28 @@ export async function hasDiff(branchName: string): Promise<boolean> {
   }
 }
 
+/**
+ * File to commit
+ */
+export interface File {
+  /**
+   * Relative file path
+   */
+  name: string;
+
+  /**
+   * file contents
+   */
+  contents: string | Buffer;
+}
+
+type CommitFilesConfig = {
```

Comment:
```suggestion
export type CommitFilesConfig = {
```

as we use this type in exported function declaration